simulation/traffic_simulator/include/traffic_simulator/api/configuration.hpp

```diff
@@ -85,10 +85,9 @@ struct Configuration
         "The map_path must be a directory (given an ", std::quoted(map_path.string()), ")");
     } else if (not contains(map_path, ".osm")) {
       throw common::SimulationError("The map_path must contain at least one *.osm file");
+    } else if (not contains(map_path, ".pcd")) {
+      throw common::SimulationError("The map_path must contain at least one *.pcd file");
     }
-    // else if (not contains(map_path, ".pcd")) {
-    //   throw common::SimulationError("The map_path must contain at least one *.pcd file");
-    // }
     else {
```

Comment:
Please change the style according to the error message on https://github.com/tier4/scenario_simulator_v2/actions/runs/4072331531/jobs/7014974239.